plugin/trino-phoenix/src/main/java/io/trino/plugin/phoenix/PhoenixClient.java

```diff
@@ -484,15 +485,19 @@ public JdbcOutputTableHandle beginCreateTable(ConnectorSession session, Connecto
                 table = table.toUpperCase(ENGLISH);
             }
             schema = toPhoenixSchemaName(schema);
-            LinkedList<ColumnMetadata> tableColumns = new LinkedList<>(tableMetadata.getColumns());
+            List<ColumnMetadata> tableColumns = tableMetadata.getColumns().stream()
+                    .filter(not(ColumnMetadata::isHidden))
```

Comment:
Do we need this check ?

Reply:
Yes. This pr introduces metadata for `ROWKEY` too. As `ROWKEY` is built in column it is hidden.